counscious_design.md

```diff
@@ -0,0 +1,18 @@
+## Conscious Design
+
+Minecraft is a game for all ages, while this does not nescessarily mean that mods shouldn't include violence, you must still keep in mind that children will
+unconsciously see what you make as a blocky representation of the outlines of reality. With that in mind, you must be conscious of a few things:
+
+- Resources are not unlimited, much like on Earth. For example, let us suppose a tech mod implements hydroelectricity - such a tech mod should also make
```

Comment:
I quite like this. Minecraft survival (no cheats) does a very good job at this. It shows that the cost of mining is a deformed landscape. Of course, there's still much world left. However, the spawn area is basically fucked.